modules/distribution/gke/main.tf

```diff
@@ -70,29 +64,3 @@ resource "google_container_node_pool" "primary_nodes" {
     }
   }
 }
-
-data "google_client_config" "default" {}
-
-resource "local_file" "kube_config_yaml" {
```

Comment:
I had to have a reference in the recipe main to get the dependencies correct.
Otherwise, the provider initializes a value that does not exist when executing the deploy command and does not update it when it is actually created.
By doing so, the provider is initialized only when the file exists in the recipe path.
This code wasn't there when the deploy command was longer and I manually targeted the execution order.